TestbedScripts/TestEnvUtils.ps1

```diff
@@ -249,15 +322,14 @@ function Initialize-SimpleEnvironment {
         Write-Host "Yes."
     }
 
-    Run-Agent
+    Initialize-ForwardingRules -MacPhysical $MacPhysical -MacHNSTransparent $MacHNSTransparent -Mac1 $Mac1 -Mac2 $Mac2 -Container1IfName $Container1IfName -Container2IfName $Container2IfName
 }
 
 # Removes test environment created by Initialize-SimpleEnvironment.
 # Relies on environment variables (indirectly) - see
-# Remove-DockerNetworkAccordingToEnv, Stop-Agent, Disable-VRouterExtension.
+# Stop-Agent, Remove-DockerNetworkAccordingToEnv.
 function Remove-SimpleEnvironment {
     Write-Host "Removing simple test environment..."
-    Remove-DockerNetworkAccordingToEnv
     Stop-Agent
-    Disable-VRouterExtension
+    Remove-DockerNetworkAccordingToEnv

```

Comment:
Then I think there should be a "Cleanup Env" function, that detects if any component is running, and cleans it up if it does. That way we can call it after every single test. 

But it's not a bug, so I will approve this